crm_automated_activity/models/automated_activity.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2021 Eder Brito
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AutomatedActivity(models.Model):
+    _name = "automated.activity"
```

Comment:
@rvalyi is better crm.automated.activity ?

Reply:
@marcelsavegnago I changed this name to **crm.automated.activity**